pkgs/applications/audio/zrythm/default.nix

```diff
@@ -220,16 +216,16 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     gappsWrapperArgs+=(
-      --prefix GSETTINGS_SCHEMA_DIR : "$out/share/gsettings-schemas/${pname}-${version}/glib-2.0/schemas/"
+      --prefix GSETTINGS_SCHEMA_DIR : "$out/share/gsettings-schemas/${finalAttrs.pname}-${finalAttrs.version}/glib-2.0/schemas/"
       --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS:${breeze-icons}/share"
     )
   '';
 
   meta = with lib; {
     homepage = "https://www.zrythm.org";
     description = "Automated and intuitive digital audio workstation";
-    maintainers = with maintainers; [ tshaynik magnetophon yuu ];
+    maintainers = with maintainers; [ tshaynik magnetophon yuu astavie ];
     platforms = platforms.linux;
```

Comment:
Could add `broken = stdenv.isDarwin;` and change this line to `platforms.unix` (like what's done [here](https://github.com/NixOS/nixpkgs/blob/7bb2ccd8cdc44c91edba16c48d2c8f331fb3d856/pkgs/applications/audio/sonobus/default.nix#L83-L84)) because zrythm runs on macos but we don't have the framework to build it written.